properties/basic.py

```diff
@@ -701,11 +706,12 @@ def validate(self, instance, value):
             self.error(instance, value)
         value = self.wrapper(value)
         if isinstance(value, np.ndarray):
-            if (value.dtype.kind == 'i' and
-                    len(set(self.dtype).intersection(integer_types)) == 0):
+            if value.dtype.kind == 'i' and int not in self.dtype:
```

Comment:
This looks good - implemented on other branch: https://github.com/3ptscience/properties/pull/84/commits/de45f7c5f7c8ddbb9ac783867e4222b445249006